cmd/azmount/README.md

```diff
@@ -26,13 +25,10 @@ The way the program works is:
 
 - It uses FUSE to expose the remote file as a local file.
 
-- Whenever the program gets a read request from the kernel, it checks if that
-  part of the file is in the local cache of blocks. If it isn't, it fetches it
-  from Azure Blob Storage and saves it to the cache.
+- Whenever the program gets a read request from the kernel, it checks if that part of the file is in the local cache of blocks.
+  If it isn't, it fetches it from Azure Blob Storage and saves it to the cache.
```

Comment:
I know this isn't part of this PR, but there are a lot of "it"s in this sentence